pyanaconda/ui/gui/spokes/user.py

```diff
@@ -146,16 +146,14 @@ def apply(self):
         # changes in the username.
 
         if self._cUid.get_active():
-            self.user.uid = int(self._uid.get_value())
+            self.user.set_uid(int(self._uid.get_value()))
         else:
-            self.user.uid = 0
-            self.user.uid_mode = constants.ID_MODE_USE_DEFAULT
+            self.user.set_uid(None)
 
         if self._cGid.get_active():
-            self.user.gid = int(self._gid.get_value())
+            self.user.set_gid(int(self._gid.get_value()))
         else:
-            self.user.gid = 0
-            self.user.gid_mode = constants.ID_MODE_USE_DEFAULT
+            self.user.set_gid(None)
```

Comment:
Just wonder if instead of setting `None` if we shouldn't have another setter. I mean something like `set_empty_gid()`.

Reply:
Is that worth the work? There are two places where this could be used instead of calling with `None` explicitly. In 8 other places it just happens depending on what comes via a variable, so there is nothing really saved with such a change.

Reply:
FWIW, :+1 to setting it to `None` explicitly (as it does here)